src/index.ts

```diff
@@ -71,6 +72,12 @@ async function bootstrap() {
     // Setup middleware
     app.use(morgan("combined"))
 
+    // Make sure we're using SSL
+    if (process.env.NODE_ENV !== "development") {
```

Comment:
We are already importing a few vars from `process.env` at the top, can import `NODE_ENV` there as well